paddle/fluid/framework/operator.cc

```diff
@@ -1007,21 +1007,23 @@ static void CheckTensorNANOrInf(const std::string& op_type,
                               op_type, name));
 }
 
-bool OperatorWithKernel::SupportsMKLDNN() const {
+bool OperatorWithKernel::SupportsMKLDNN(const framework::ExecutionContext& ctx,
```

Comment:
The `framework::ExecutionContext& ctx` is not used in this function. Is it really needed?

Reply:
Done